ga4gh/datamodel/reads.py

```diff
@@ -68,6 +68,8 @@ class SamFlags(object):
     """
     NUMBER_READS = 0x1
     PROPER_PLACEMENT = 0x2
+    UNMAPPED = 0x4
+    NEXT_SEGMENT_UNMAPPED = 0x8
```

Comment:
@calbach another question(s):

In the migration guide, the attributes `read.alignment.position.position` and `read.nextMatePosition.position` are tested for a null value.  However, the avro `Position` record does not allow null values for the `position` attribute.  Is this an error in the schema, or an error in the migration guide?  If the former, 1) should a read's `position` and `nextMatePosition` not be allowed to be null and 2) for either attribute if the `position` attribute is null, what should the `Position`'s `referenceName` and `strand` be set to?